quantize.py

```diff
@@ -404,7 +403,7 @@ def __init__(self, mod, groupsize=128, inner_k_tiles=8, padding=True):
 
     @torch.no_grad()
     def create_quantized_state_dict(self, use_cuda = True):
-        if use_cuda:
+        if use_cuda and torch.cuda.is_available():
```

Comment:
won't be necessary to add `torch.cuda.is_available()`, just let it report error if `use_cuda` is true and gpu is not available.